manifests/init.pp

```diff
@@ -35,12 +36,11 @@
       fail("Unsupported OS: ${::operatingsystem}")
     }
   }
-
   # Build full list of features to install
   $features = union($default_features, $extra_features)
 
   class { '::opendaylight::install': } ->
-  class { '::opendaylight::config': } ~>
+  class { '::opendaylight::config': odl_rest_port => $odl_rest_port} ~>
```

Comment:
This is throwing a style check error (alignment of arrows): https://travis-ci.org/dfarrell07/puppet-opendaylight/jobs/49525399